server/server.js

```diff
@@ -22,6 +21,19 @@ app.get('/getData/:itemId', (request, response) => {
     },request.params.itemId)
 })
 
+app.get('/search/:searchString', (request, response) => {
+    db.searchProducts((err, results) => {
+        if (err) {
+            console.log('GETPRODUCTDATAERROR: ', err);
```

Comment:
Do you want to response.send the error, or just console log it?

Reply:
Nice, moved it outside the db method.

Reply:
Actually, moving it outside the db method causes the code to break